modules/videopress/videopress.php

```diff
@@ -22,7 +22,7 @@ public static function init() {
 
 	function __construct() {
 		$this->version = time(); // <s>ghost</s> cache busters!
-		add_action( 'jetpack_modules_loaded', array( $this, 'jetpack_modules_loaded' ) );
+		add_action( 'init', array( $this, 'jetpack_modules_loaded' ) );
```

Comment:
Good idea. I force-pushed a commit to change that function name and clarify the inline comment. @zinigor all good?